src/Network.java

```diff
@@ -290,7 +305,51 @@ public void backPropagate(List<Double> target) {
         }
     }
 
-    public List<Double> calculateError() {return null;}
+    /**
+     * Converts the network to a chromosome
+     * @return Returns the network represented as chromosome
+     */
+    public Chromosome toChromosome() {
+        List<List<Node>> multiNodes = new ArrayList<>();
+        for(Layer layer : this.layers){
+            multiNodes.add(layer.nodes);
+        }
+        List<Node> nodes = multiNodes.
+                parallelStream().
+                flatMap(Collection::stream).
+                collect(Collectors.toList());
+        IntStream.
+                range(0,nodes.size()).
+                parallel().
+                forEach(index -> nodes.get(index).id = index);
+        double[][] adjacencyMatrix = new double[nodes.size()][nodes.size()];
+        nodes.
+                stream().
+                parallel().
+                forEach(
+                        (j) ->
```

Comment:
There has to be a way to format this better?  Maybe something closer to [Line 344](https://github.com/Penchant/EvolutionaryAlgorithms/pull/10/files#diff-6b0a88b476e0e20150b12df69d3b7e9eR344)